### PR TITLE
Active Record Basics && Assets and Navigation: Rewrite headings to follow sentence case Style Guide

### PR DESCRIPTION
**Pull request by danieletie**
## This PR rewrites the headings in the sentence case style guide for

- active_active_records lesson
- project_micro_reddit
- asset_pipeline
- importmap
- turbo_drive

## Issue

Related to #26169

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by danieletie** [comment]
Hello @ManonLef could you please take a look to make sure everything is right. Like I mentioned, it's my first contribution. I tried as much as possible to follow everything in the docummntations.

**Review by wise-king-sullyman — APPROVED** [review]
This looks great, thank you!

**Comment by danieletie** [comment]
> This looks great, thank you!

It was my pleasure. Do you reckon I can get assigned some other lessons?

File: ruby_on_rails/assets_and_navigation/turbo_drive.md
Comment (ManonLef) [inline ruby_on_rails/assets_and_navigation/turbo_drive.md:1]: This lesson still has a header: `Application Visit` where `Visit` needs to start with a lowercase `v` :) Would you like to make a PR for that @danieletie ?

Comment (danieletie) [reply ruby_on_rails/assets_and_navigation/turbo_drive.md:1]: sure

**Pull request merged**